suite-native/app/e2e/tests/deeplinkPopup.test.ts

```diff
@@ -0,0 +1,165 @@
+import http from 'http';
+import { exec } from 'child_process';
+
+import { TrezorUserEnvLink } from '@trezor/trezor-user-env-link';
+import TrezorConnect from '@trezor/connect-mobile';
+
+import {
+    appIsFullyLoaded,
+    disconnectTrezorUserEnv,
+    openApp,
+    prepareTrezorEmulator,
+    restartApp,
+} from '../utils';
+import { onOnboarding } from '../pageObjects/onboardingActions';
+import { onCoinEnablingInit } from '../pageObjects/coinEnablingActions';
+import { onHome } from '../pageObjects/homeActions';
+
+const SERVER_PORT = 8080;
+const SERVER_URL = `http://localhost:${SERVER_PORT}`;
+
+let server: http.Server | undefined;
+
+const openUriScheme = (url: string, platformToOpen: 'android') => {
```

Comment:
Use the [detox device API](https://wix.github.io/Detox/docs/guide/mocking-open-with-url) instead, please.

Reply:
It sounds like good idea but I have been testing it and does not really works as this test need, so I would keep this approach.